lib/sockets/pbpal_resolv_and_connect_sockets.c

```diff
@@ -148,7 +155,7 @@ static void get_dns_ip(struct sockaddr* addr)
             && (pubnub_get_dns_secondary_server_ipv6((struct pubnub_ipv6_address*)pv6)
                 == -1)) {
             addr->sa_family = AF_INET;
-            inet_pton(AF_INET, PUBNUB_DEFAULT_DNS_SERVER, p);
+            get_default_dns_ip((struct pubnub_ipv4_address*)p);
```

Comment:
if statement is checking with `(struct pubnub_ipv6_address*)pv6` but inside that code is using` (struct pubnub_ipv4_address*)p` . Can you please clarify, if that is ok.

Reply:
Shouldn't that be `get_default_dns_ip((struct pubnub_ipv6_address*)pv6);` ?

Reply:
Same answer here as on line 129. The code expects IPv4 here, due to setting the address family.